src/datomic_rest_api/rest/gene.clj

```diff
@@ -533,12 +533,13 @@
         phenos (set (concat (keys var-phenos)
                             (keys rnai-phenos)))]
     (->>
-      (for [pid phenos :let [pheno (entity db pid)]]
-        (let [pcs (get-pato-combinations db pid rnai-phenos var-phenos not?)]
-          (if (nil? pcs)
-            (phenotype-table-entity db pheno nil nil pid var-phenos rnai-phenos not?)
-            (for [[pato-key entity] pcs]
-              (phenotype-table-entity db pheno pato-key entity pid var-phenos rnai-phenos not?)))))
+      (flatten
+        (for [pid phenos :let [pheno (entity db pid)]]
```

Comment:
I think the style convention for `:let` in forms brings it onto its own line e.g:
```clojure
(for [x xs
      :let [n-xs (count xs)]]
    (do-something x))
```